src/index.coffee

```diff
@@ -3,8 +3,8 @@ Path  = require('path')
 Hubot = require('hubot')
 
 class MockRobot extends Hubot.Robot
-  constructor: ->
-    super null, null, false, 'hubot'
+  constructor: (httpd=false) ->
```

Comment:
`super null, null, false, 'hubot'` had me believe that for *this* testing you wanted the server to be off.

I can make those changes this weekend.